fedocal/api.py

```diff
@@ -428,3 +428,47 @@ def callback(response):
         status=status,
         mimetype='application/json'
     )
+
+
+@APP.route('/api/<username>/shield/<calendar_name>/')
+@APP.route('/api/<username>/shield/<calendar_name>')
+def api_user_shield(username, calendar_name):
+    """
+User shield
+===========
+
+Provides a small image (a shield) displaying the status of the specified
+user on the specified calendar if the user is currently managing a meeting.
+
+If the user is not managing a meeting, this endpoint does not return
+anything.
+
+    """
+    @flask.after_this_request
+    def callback(response):
+        """ Handle case the query was an JQuery ajax call. """
+        return check_callback(response)
+
+    calendarobj = Calendar.by_id(SESSION, calendar_name)
+    if not calendarobj:
+        flask.abort(400, 'Invalid calendar provided')
+
+    start_date = datetime.datetime.utcnow().date()
+    end_date = start_date + datetime.timedelta(days=1)
+
+    meetings = fedocallib.get_by_date(
+        SESSION, calendarobj, start_date, end_date, tzone='UTC')
+
+    output = {'status': ''}
```

Comment:
> I thought green looks nicer :)

Oh, it does.  I meant red if you are *not* on vacation.

Now that I look at the pkgdb2 screenshot though, having just one icon looks fine.

Perhaps, return ``None`` instead of the empty string ``''`` when they're not on vacation.